src/Controller/SponsorshipLevel/Create.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\SponsorshipLevel;
+
+use App\Form\SponsorshipLevelType;
+use App\Service\DTO\SponsorshipLevelData;
+use App\Service\Manager\SponsorshipLevelManagerInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig_Environment;
+
+final class Create
+{
+    private $renderer;
+    private $sponsorshipLevelManager;
+    private $formFactory;
+    private $router;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SponsorshipLevelManagerInterface $sponsorshipLevelManager,
+        FormFactoryInterface $formFactory,
+        RouterInterface $router
+    )
+    {
+        $this->renderer = $renderer;
+        $this->sponsorshipLevelManager = $sponsorshipLevelManager;
+        $this->formFactory = $formFactory;
+        $this->router = $router;
+    }
+
+    /**
+     * @param Request $request
+     * @return Response
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function handle(Request $request): Response
+    {
+        $sponsorshipLevelData = new SponsorshipLevelData();
+
+        $form = $this->formFactory->create(SponsorshipLevelType::class, $sponsorshipLevelData);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $sponsorshipLevelData->setPosition((int) $this->sponsorshipLevelManager->getMaxPosition() + 1);
```

Comment:
It's the second time I can see this line in this PR, either create an incrementPosition method that does that, or do something else you like, but change that please.